lib/contentful_middleman/helpers.rb

```diff
@@ -28,10 +29,11 @@ def localize_array(value, locale, fallback_locale='en-US')
     end
 
     def localize_value(value, locale, fallback_locale='en-US')
-      if value.respond_to? :fetch
-        return value.fetch(locale) if value.key? locale
-        return value.fetch(fallback_locale) if value.key? fallback_locale
-      end
+      value = value.fetch(locale) if value.respond_to?(:fetch) && value.respond_to?(:key) && value.key?(locale)
```

Comment:
I changed it to `value.respond_to?(:key?)`.

The issue I was trying to avoid with this is if `value` is passed an `Array` (shouldn't happen aside from user error?) or `value` becomes an `Array` via `value = value.fetch(locale)`, `Array`s do have `fetch` on them, but not `key?`, so it would throw a `NoMethodError` if it matched on the locale key.

Not sure which syntax is more ruby-like – I'd write this differently in Javascript:
```
  if value.respond_to?(:fetch) && value.respond_to?(:key?)
    if value.key?(locale)
      value = value.fetch(locale)
    elsif value.key?(fallback_locale)
      value = value.fetch(fallback_locale)
    end
  end
```
or
```
  value = value.fetch(locale) if value.respond_to?(:fetch) && value.respond_to?(:key?) && value.key?(locale)
  value = value.fetch(fallback_locale) if value.respond_to?(:fetch) && value.respond_to?(:key?) && value.key?(fallback_locale)
```

I re-committed the more succinct syntax.